README.markdown

```diff
@@ -87,6 +91,14 @@ For Windows:
     -L+org.eclipse.swt.win32.win32.x86.lib -L+dwt-base.lib -L/SUBSYSTEM:WINDOWS:4.0
   ```
 
+  If you use Microsoft linker:
+
+  ```
+  $ dmd main.d -I<dwt>\imp -J<dwt>\res -L/LIBPATH:<dwt>\lib ^
```

Comment:
`-J<dwt>\res` is required for compiler. dmd reads `res\org.eclipse.swt.internal.SWTMessages_*.properties` when compile time.